### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my card components

this page contains the card components of Component library

live link: https://componentsofcss.netlify.app/component%20html/card.html

Following are the features of this page

nav: contains brand name and links to the home page and components library.
main: contains the documentation of card components with snippets and examples.
aside: contains links to all my component libraries.
footer: contains links to my social media

image

### CODE REVIEW

File: utkarsh component/card/card.css [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+}
+li{
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+li a{
+    /* text-decoration: none; */
```
Comment (khushijohri2001) [inline utkarsh component/card/card.css]: Remove the unwanted comment here to make the code cleaner and readable.

File: utkarsh component/card/card.css [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+}
+
+a:hover{
+    color: blue;
+  
+}
+
+main{
+    
+    /* grid-template-areas: 200px 1fr; */
```
Comment (khushijohri2001) [inline utkarsh component/card/card.css]: Remove the unwanted comment here to make the code cleaner and readable.

File: utkarsh component/card/card.css [hunk 3]
```diff
@@ -0,0 +1,599 @@
...
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+
+h3{
+    font-size: 1.5rem;
+    /* line-height: 2rem; */
```
Comment (khushijohri2001) [inline utkarsh component/card/card.css]: here too

**Review by khushijohri2001 — APPROVED** [review]

File: utkarsh component/card/card.css [hunk 4]
```diff
@@ -0,0 +1,599 @@
...
+
+footer ul{
+ padding-inline-start: 0.5rem;
+    display: flex;
+  gap: 20px;
+
+
+}
+
+
```
Comment (i-am-a-codegirl) [inline utkarsh component/card/card.css:146]: remove extra space

File: utkarsh component/card/card.css [hunk 5]
```diff
@@ -0,0 +1,599 @@
...
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+#check{
+  display: none;
+}
+@media (max-width: 925px){
+  label.logo{
+    font-size: 30px;
```
Comment (i-am-a-codegirl) [inline utkarsh component/card/card.css:362]: Please put media queries in the end of your CSS and not in between

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**